### ISSUE TITLE
Toggle: unbind event  after partial fetched

### ISSUE THREAD

**Issue by fffx** [nikolalsvk/render_async#94]
Shouldn't we unbind the  [Event](https://github.com/renderedtext/render_async#toggle-event)  after partial fetched?   I notice that the event still triggered and fetch the partial every time after the partial already be rendered.  

In jquery we can achieve this by use [namespace](https://api.jquery.com/event.namespace/) event, and in pure js we can use [removeEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener)

I would like make PR if this is a bug?

**Comment by nikolalsvk** [comment]
I am not sure about unbinding the event, idea is to possibly toggle `render_async` as many times as you want. For example, maybe users want to do something like this:
 
```erb
<a href='#' id='reload-comments'>Reload</a>
<%= render_async comments_path, toggle: { selector: '#reload-comments', event: :click } %>
```

But, we can add a possibility to unbind the event if it's a problem for some folks. Leave 👍 if you'd like to see this in `render_async`

**Comment by Tab10id** [comment]
> For example, maybe users want to do something like this

I want it. I'd love to render some widget by additional request and rerender it when some event happened. For now I can achive this only by hack with `container_id` (I render wrapper with that id on renderred partial).

**Comment by nikolalsvk** [comment]
This is solved in https://github.com/renderedtext/render_async/pull/96 and released in 2.1.4 version https://github.com/renderedtext/render_async/releases/tag/2.1.4
